### PR TITLE
Contents + Metadata only needs paths, so we don't need to provide the whole service.

### PR DESCRIPTION
**Pull request by cbeer**
The pattern of sending the whole service to the ~10 supporting actions + classes makes it hard for me to understand what's going on. It looks like some of these only need the paths helper, and maybe that will help my understanding.

### CODE REVIEW

File: app/services/versioned_files_service.rb [hunk 1]
```diff
@@ -21,12 +21,9 @@ def self.versioned_files?(druid:)
   def initialize(druid:)
     @druid = druid
     @paths = Paths.new(druid:)
-    @version_manifest = VersionsManifest.new(path: @paths.versions_manifest_path)
-    @contents = Contents.new(service: self)
-    @metadata = Metadata.new(service: self)
   end
```
Comment (mjgiarlo) [inline app/services/versioned_files_service.rb:24]: Do you think giving line 23 the same treatment is worth considering? (i.e., extracting the ivar into a new instance method that memoizes the same ivar)

Comment (mjgiarlo) [reply app/services/versioned_files_service.rb:24]: Oh, nevermind. I now see the other extracted methods depend on `@paths` already.

Comment (cbeer) [reply app/services/versioned_files_service.rb:24]: I thought about it, but would rather do that in a separate PR if we think it's a good idea.

**Comment by justinlittman** [comment]
Approve pending resolving the conflict.

**Pull request merged**